target/linux/ramips/mt7620/base-files/etc/board.d/02_network

```diff
@@ -301,6 +306,11 @@ ramips_setup_macs()
 		[ -n "$lan_mac" ] || lan_mac=$(cat /sys/class/net/eth0/address)
 		wan_mac=$(macaddr_add "$lan_mac" 1)
 		;;
+	hootoo,tm05)
+		lan_mac=$(mtd_get_mac_binary factory 0x4)
+		wan_mac=$(mtd_get_mac_binary factory 0x2e)
+		label_mac=$lan_mac
```

Comment:
Use label-mac-device alias in DTS instead.

Reply:
So just the last line (label), use this instead in DTS (wmac => lan_mac)?
```
	aliases {
		label-mac-device = &wmac;
	};
```

Reply:
Just remove the label_mac line here.
You would then need 
```
aliases {
	label-mac-device = &ethernet;
};
```
in DTS.